addon/components/detail.js

```diff
@@ -138,11 +152,27 @@ export default Component.extend(PropTypeMixin, {
     return _.cloneDeep(bunsenView)
   },
 
+  /**
+   * Precompute all model dependencies
+   * @param {Object[]} cells - the view cells
+   * @param {Object} cellDefinitions - the view cell definitions
+   * @returns {Object} cellConfigs with  precomputed model dependencies
+   */
   @readOnly
-  @computed('renderView')
-  cellTabs () {
-    const cells = this.get('renderView.cells')
+  @computed('renderView.cells', 'renderView.cellDefinitions')
+  precomputedCellConfig (cells, cellDefinitions) {
```

Comment:
So we are now pre-dereferencing the cells instead of the using `mergedConfig` logic I had in the cell component?